scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/MerlinService.java

```diff
@@ -54,7 +58,7 @@ Plan getPlanActivities(final PlanMetadata planMetadata, final MissionModelWrappe
    * @return the database id of the newly created aerie plan container
    * @throws NoSuchPlanException when the plan container could not be found in aerie after creation
    */
-  PlanId createNewPlanWithActivities(final PlanMetadata planMetadata, final Plan plan)
+  Pair<PlanId, Map<ActivityInstance, ActivityInstanceId>> createNewPlanWithActivities(final PlanMetadata planMetadata, final Plan plan)
```

Comment:
because when we get the plan back from the scheduler and we want to compute the results, you get a collection of activity instances and have to find which merlin activity instance it corresponds to.